Cloudinary

directtoanne opened the pull request.

JakeM93 commented on .gitignore:
```diff
@@ -41,3 +41,4 @@ yarn-debug.log*
 *.swp
 .DS_Store
 .env*
+.env*
```
is this duplicate line?

JakeM93 commented:
This was working only locally as the details in the .env are only local to the person who set it up, so this may only work for you.

directtoanne commented:
> This was working only locally as the details in the .env are only local to the person who set it up, so this may only work for you.

maybe... idk i can't really test myself non locally but i added a cloudinary.yml file in the config and all the image tags for bookings , index , my cars etc are now pulling from cloudinary rather than the placeholder.

meanwhile also added delete functions for booking and cars.
